pkg/terminal/command.go

```diff
@@ -1840,6 +1840,7 @@ func setBreakpoint(t *Term, ctx callContext, tracepoint bool, argstr string) ([]
 		if match := r.FindStringIndex(argstr); match != nil {
 			requestedBp.Name = ""
 			cond = argstr[match[1]:]
+			requestedBp.Cond = cond
```

Comment:
Assign `argstr[match[1]:]` directly to `requestedBp.Cond` and delete the `cond` variable.